src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int i = new Random().nextInt(Color.values().length);
```

Comment:
Don't create Random instance every time you call getRandomColor, better to make it constant.
Also, better to use SecureRandom instead.